common/changes/@binaris/shift-babel-macro/master_2019-07-02-12-37.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@binaris/shift-babel-macro",
+      "comment": "Add clean command",
+      "type": "patch"
```

Comment:
I think this can be a `"type": "none"` change (`rush change` and Enter without describing changes) since it doesn't affect outside users, and relates only to the build process and won't add noise to the changelog.